schemas/ispyb/stored_programs/sp_retrieve_dc.sql

```diff
@@ -11,13 +11,15 @@ BEGIN
 				SELECT dc.dataCollectionId "id",
 					dc.dataCollectionGroupId "groupId",
 					dc.detectorId "detectorId",
+					dc.BLSAMPLEID "blSampleId",
 					dc.blSubSampleId "blSubSampleId",
 					dc.dataCollectionNumber "dcNumber",
 					dc.APERTUREID "apertureId",
 					dc.POSITIONID "positionId",
+					dc.screeningOrigId "screeningOrigId",
 					dc.startTime "startTime",
 					dc.endTime "endTime",
-					dc.runStatus "runStatus",
+					dc.runStatus "status",
```

Comment:
Sorry, we can't rename this alias.

Reply:
Yeah I think it is supposed to be `status` according to the master branch. I don't know why it went back to `runStatus` on my end but this is where I'm correcting it.

Reply:
Ah, so it was status, then accidentally got changed. I see. Then it's fine.